Source/Layout/ASLayoutElement.mm

```diff
@@ -150,7 +150,8 @@ @implementation ASLayoutElementStyle {
   std::atomic<CGPoint> _layoutPosition;
 
 #if YOGA
-  std::atomic<ASStackLayoutDirection> _direction;
+  std::atomic<ASStackLayoutDirection> _flexDirection;
+  std::atomic<YGDirection> _direction;
```

Comment:
As all of them are currently c++ atomics let's go with it for now and do the whole change in a another diff if we want to.